geti_sdk/geti.py

```diff
@@ -193,6 +203,33 @@ def __init__(
         # Cache of deployment clients for projects in the workspace
         self._deployment_clients: Dict[str, DeploymentClient] = {}
 
+    def _check_platform_version(self) -> None:
+        """
+        Check the version of the Intel® Geti™ server that this `Geti` instance is
+        connected to. If the version is not supported by the SDK, a warning will be
+        issued.
+
+        :raises: ValueError if the Intel® Geti™ server version is not supported by the
+            Intel® Geti™ SDK.
+        """
+        # Get the build version without a timestamp
+        platform_version = self.session.version.version
```

Comment:
Minor suggestion for another PR: repetitions like `.version.version` aren't a good software practice; the problem is the `GetiVersion` class which exposes a `version` attribute. I believe it would be better if that information was exposed through `__str__`, so that `str(session.version)` would return a string like "1.8.0". Methods like `__eq__`, `__lt__`, ... could be be smarter and support both strings and `GetiVersion` objects.

Reply:
For the `__str__` method - what you suggested is what is already implemented in the code.
A `GetiVersion` takes into account both the release version and the timestamp, while it's  `version` field only represents the numeric version.
May be we should rename it to `base_version`